### PR TITLE
new package: py-ics

### PR DESCRIPTION
**Pull request by healther**
depends on #13575, #13566

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-ics/package.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    Ics.py is available for Python>=3.6 and is Apache2 Licensed.
+    """
+
+    homepage = "https://github.com/C4ptainCrunch/ics.py"
+    git      = "git@github.com:C4ptainCrunch/ics.py.git"
+
+    # for some reason only a .egg version is published on pypi and
+    # github's archive link doesn't fetch for some reason, so use
+    # the commit for the associated release
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ics/package.py]: What do you mean the GitHub archive link wouldn't fetch? It fetches for me.

File: var/spack/repos/builtin/packages/py-ics/package.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    version('0.6', commit='74d7b24cb1639d2cd027e502d5f16ea68b5e5b12')
+    version('0.5', commit='8ac7db18d365d74ce6524a441a4c4548df3349ee')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-python-dateutil', type=('build', 'run'))
+    depends_on('py-arrow@0.11:0.14.99', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ics/package.py]: Version 0.5 has a different dependency range. Honestly, you could just not add 0.5 and make the package a lot simpler.

File: var/spack/repos/builtin/packages/py-ics/package.py (hunk 1 above)
Comment (healther) [reply var/spack/repos/builtin/packages/py-ics/package.py]: Actually I'm not sure anymore, either way fetching works with 84daf6a

File: var/spack/repos/builtin/packages/py-ics/package.py (hunk 2 above)
Comment (healther) [reply var/spack/repos/builtin/packages/py-ics/package.py]: done

File: var/spack/repos/builtin/packages/py-ics/package.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    Ics.py is available for Python>=3.6 and is Apache2 Licensed.
+    """
+
+    homepage = "https://github.com/C4ptainCrunch/ics.py"
+    url      = "https://github.com/C4ptainCrunch/ics.py/archive/v0.5.tar.gz"
+
+    version('0.6', sha256='4947263136202d0489d4f5e5c7175dfd2db5d3508b8b003ddeaef96347f68830')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ics/package.py]:
```suggestion
    depends_on('py-setuptools', type='build')
```

File: var/spack/repos/builtin/packages/py-ics/package.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    calendars.
+
+    iCalendar (file extension .ics) is used by Google Calendar, Apple
+    Calendar, Android and many more.
+
+    Ics.py is available for Python>=3.6 and is Apache2 Licensed.
+    """
+
+    homepage = "https://github.com/C4ptainCrunch/ics.py"
+    url      = "https://github.com/C4ptainCrunch/ics.py/archive/v0.5.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ics/package.py]:
```suggestion
    url      = "https://github.com/C4ptainCrunch/ics.py/archive/v0.6.tar.gz"
```

File: var/spack/repos/builtin/packages/py-ics/package.py (hunk 3 above)
Comment (healther) [reply var/spack/repos/builtin/packages/py-ics/package.py]: good catch

File: var/spack/repos/builtin/packages/py-ics/package.py (hunk 4 above)
Comment (healther) [reply var/spack/repos/builtin/packages/py-ics/package.py]: Done, but is this still required? Spack seems to be able to handle anything that follows a typical versioning scheme

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-ics/package.py]: It's not required for downloading, but the `spack url summary` command will report inaccurate metrics if the URL does not match at least one version in the package.

**Pull request merged**